modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -191,6 +212,6 @@
         'jarsigner_key_store' => $jarsigner_keystore,
         'jarsigner_certificate_aliases' => $jarsigner_certificate_aliases_content,
 
-        'taskcluster_scope_prefix' => $_env_config['scope_prefix'],
+        'taskcluster_scope_prefix' => $_env_config['scope_prefix']
```

Comment:
Nit: No need to strip the comma at the end. Hiera hashes are smarter than JSON objects.